Cloud testapp

375449 opened the pull request:
ДЗ к занятию №6

### Данные для подключения:
testapp_IP = 104.155.8.119
testapp_port = 9292

### Создание ВМ с автозапуском скрипта
gcloud compute instances create reddit-app 
--boot-disk-size=10GB
--image-family ubuntu-1604-lts 
--image-project=ubuntu-os-cloud 
--machine-type=g1-small 
--tags puma-server 
--restart-on-failure 
--metadata-from-file startup-script=375449_infra/startup_script.sh 
--zone europe-west1-b

### Создание правила файервола посредством команды gcloud
gcloud compute firewall-rules create default-puma-server 
--network default 
--action allow 
--direction ingress 
--rules tcp:9292 
--source-ranges 0.0.0.0/0 
--priority 1000 
--target-tags puma-server

375449 committed:
Move two files, HW 06

375449 committed:
HW 06

375449 committed:
HW 06

375449 committed:
HW 06

375449 committed:
HW 06

mrgreyves reviewed (changes requested):
Здравствуйте!) Проверил ваше дз. Постарайтесь починить тесты. Вполне возможно что во время тестов вы уже выключили вм. Пока возвращаю на доработку.

375449 committed:
Delete startup_script.sh

375449 committed:
Delete install_ruby.sh

375449 committed:
Delete install_mongodb.sh

375449 committed:
Delete deploy.sh

375449 committed:
HW 06

375449 committed:
git commit
Merge branch 'cloud-testapp' of https://github.com/otus-devops-2019-02/375449_infra into cloud-testapp

ddsf

t
^:quit
Qd     ially if it merges an updated upstream into a topic branch.

375449 committed:
HW 06

375449 committed:
HW 06

mrgreyves reviewed (approved):
Принимаю!=)
